_api-reference/script-apis/exec-stored-script.md

```diff
@@ -22,13 +21,13 @@ OpenSearch provides several ways to run a script; the following sections show ho
 
 #### Sample request
 
-The following request runs the stored script that was created in [Create or Update Stored Script]({{site.url}}{{site.baseurl}}/opensearch/rest-api/_script-apis/create-stored-script/). The script sums the ratings for each book and displays the sum in the `total_ratings` field in the output.
+The following request runs the stored script that was created in [Create or Update Stored Script]({{site.url}}{{site.baseurl}}/api-reference/script-apis/create-stored-script/). The script sums the ratings for each book and displays the sum in the `total_ratings` field in the output.
 
 * The script's target is the `books` index.
 
 * The `"match_all": {}` property value is an empty object indicating to process each document in the index.
 
-* The `total_ratings` field value is the result of the `my-first-script` execution. (See  [Create or Update Stored Script]({{site.url}}{{site.baseurl}}/opensearch/rest-api/_script-apis/create-stored-script/).)
+* The `total_ratings` field value is the result of the `my-first-script` execution. (See  [Create or Update Stored Script]({{site.url}}{{site.baseurl}}/api-reference/script-apis/create-stored-script/).)
```

Comment:
Link should use sentence case to match the linked heading.

`[Create or update stored script]({{site.url}}{{site.baseurl}}/api-reference/script-apis/create-stored-script/)`

There's also an extra space between "See" and the link itself.

Reply:
Also the nested parentheses should be taken behind the barn. Should just end with:

```
See [Create or update stored script]({{site.url}}{{site.baseurl}}/api-reference/script-apis/create-stored-script/).
```